app/common/patient/directives/patientSearch.html

```diff
@@ -0,0 +1,74 @@
+<div class="panel panel-primary content-border-margin"><!-- demographics -->
+  <div class="panel-heading">
+    <h3 class="panel-title">{{'SEARCH_PATIENT' | translate}}</h3>
+  </div>
+  <div class="panel-body">
+    <div class="row">
+      <div class="col-sm-10">
+        <div class="input-group input-group-lg">
+                    <span class="input-group-addon">
+                        <span class=" glyphicon glyphicon glyphicon-search"></span>
+                    </span>
+          <input type="text" class="form-control input input-lg keyboard-in" ng-model="vm.searchText"
+                 ng-change="vm.change()" placeholder="{{'SEARCH_PATIENT_PLACEHOLDER' | translate}}">
+        </div>
+      </div>
+      <div class="col-sm-2">
+        <poc-authorize privilege="'Write Patient'">
+          <button type="button" class="btn btn-primary btn-lg" ng-click="vm.linkPatientNew()">
+            <span class="glyphicon glyphicon-plus-sign" />
+            <span class="button-label">{{'NEW_PATIENT' | translate}}</span>
+          </button>
+        </poc-authorize>
+      </div>
+    </div>
+    <table st-table="vm.results" st-safe-src="vm.displayed" class="table table-striped">
+      <thead>
+      <tr>
+        <th>{{'SEARCH_PATIENT_NID' | translate}}</th>
+        <th st-sort="givenName">{{'SEARCH_PATIENT_FIRSTNAME' | translate}}</th>
+        <th st-sort="familyName">{{'SEARCH_PATIENT_LASTNAME' | translate}}</th>
+        <th st-sort="gender">{{'SEARCH_PATIENT_GENDER' | translate}}</th>
+        <th st-sort="age.years">{{'SEARCH_PATIENT_AGE' | translate}}</th>
+        <th st-sort="birthdate">{{'SEARCH_PATIENT_BIRTHDATE' | translate}}</th>
+        <th ng-if="vm.showSchedule">{{'SEARCH_SCHEDULE_STATUS' | translate}}</th>
```

Comment:
Pharmacy schedule is different comparing with registration and clinic. Pharmacy uses the next pickup date whereas others use the next consultation date

Reply:
I don't understand this one.

Reply:
The value of **Scheduled for** in search list for the pharmacy should be the last entry of **next pickup date** for the patient. Look for the last `obs` (concept id: `5096`, uuid: `e1e2efd8-1d5f-11e0-b929-000c29ad1d07`) for the patient. The difference with registration and clinic modules is that the value is for the last entry  (obs) of **next consultation date** for the patient, concept id: `1410`, uuid `e1dae630-1d5f-11e0-b929-000c29ad1d07`